concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+Lastra-1986-381-68	68	viruela	smallpox		
+Lastra-1986-381-69	69	su cuerno	his horn	1393	HORN (ANATOMY)
+Lastra-1986-381-70	70	pico (de un pájaro)	peak (of a bird)	73	BEAK
+Lastra-1986-381-71	71	ala	wing	1257	WING
+Lastra-1986-381-72	72	pluma	feather	1201	FEATHER
+Lastra-1986-381-73	73	molleja	gizzard		
+Lastra-1986-381-74	74	huevo	egg	744	EGG
+Lastra-1986-381-75	75	mujer	women	962	WOMAN
+Lastra-1986-381-76	76	mi esposa	my wife	1199	WIFE
+Lastra-1986-381-77	77	hombre	man	1554	MAN
+Lastra-1986-381-78	78	marido	husband	1200	HUSBAND
+Lastra-1986-381-79	79	papá	dad	1217	FATHER
+Lastra-1986-381-80	80	mamá	mother	1216	MOTHER
+Lastra-1986-381-81	81	mi hijo	my son		
+Lastra-1986-381-82	82	huérfano	orphan	1149	ORPHAN
+Lastra-1986-381-83	83	entenado (o hijastro)	stepson		
+Lastra-1986-381-84	84	padrastro	stepfather	237	STEPFATHER
+Lastra-1986-381-85	85	madrastra	stepmother	236	STEPMOTHER
+Lastra-1986-381-86	86	curandero	medicine man	394	SHAMAN (FOLK HEALER)
+Lastra-1986-381-87	87	brujo	sorcerer	2137	MAGICIAN
+Lastra-1986-381-88	88	animal (ver si hay differencia entre animal pequeño o grande, doméstico o salvaje)	Animal (see if there is differented between small or large, domestic or wild animal)	619	ANIMAL
+Lastra-1986-381-89	89	guajolote (si además distiguen por tamaño y sexo, anotar enfrente)	Guajolote (if they also distribute by size and sex, write down in front)	2386	TURKEY
+Lastra-1986-381-90	90	gallina	hen	1514	HEN
+Lastra-1986-381-91	91	gavilán	hawk	1865	BUZZARD
+Lastra-1986-381-92	92	zopilote	buzzard		
+Lastra-1986-381-93	93	tecolotes	Tecolotes		
+Lastra-1986-381-94	94	pájaro carpintero (o cherreque)	woodpecker		
+Lastra-1986-381-95	95	iguana	Iguana	1578	IGUANA
+Lastra-1986-381-96	96	lagartija	lizard	632	LIZARD
+Lastra-1986-381-97	97	conejos	rabbits		
+Lastra-1986-381-98	98	venados	deers		
+Lastra-1986-381-99	99	cerdo	pig	1337	PIG
+Lastra-1986-381-100	100	mi borrego	My sheep		
+Lastra-1986-381-101	101	armadillo	armadillo	1885	ARMADILLO
+Lastra-1986-381-102	102	ardilla	squirrel	1628	SQUIRREL
+Lastra-1986-381-103	103	mapache	raccoon	828	RACCOON
+Lastra-1986-381-104	104	tlacuache	opossum	309	OPOSSUM
+Lastra-1986-381-105	105	zorillo	skunk	3888	SKUNK
+Lastra-1986-381-106	106	muerciélago	bat	1793	BAT
+Lastra-1986-381-107	107	mariposa	butterfly	1791	BUTTERFLY
+Lastra-1986-381-108	108	pescado	fish	227	FISH
+Lastra-1986-381-109	109	ratón	mouse	1491	MOUSE
+Lastra-1986-381-110	110	rata	rat	1490	RAT
+Lastra-1986-381-111	111	perro	dog	2009	DOG
+Lastra-1986-381-112	112	mi perro	my dog		
+Lastra-1986-381-113	113	rana	frog	503	FROG
+Lastra-1986-381-114	114	sapo	toad	894	TOAD
+Lastra-1986-381-115	115	caracol	snail	1543	SNAIL
+Lastra-1986-381-116	116	gusano	worm	1219	WORM
+Lastra-1986-381-117	117	alacrán	scorpion	1538	SCORPION
+Lastra-1986-381-118	118	garrapata	tick	1527	TICK
+Lastra-1986-381-119	119	liendre	nit	267	NIT
+Lastra-1986-381-120	120	piojo (diferenciar del piojo blanco)	Piojo (differentiate from white louse)	1392	LOUSE
```

Comment:
Might need further investigation given the disambiguation in the original gloss.

Reply:
Also a bad translation on my side, `louse (differentiate from white louse)` should be it. I'd still map it as `1392 LOUSE`.